packages/data-explorer-ui/src/components/Export/components/ExportToTerra/components/ExportToTerraReady/exportToTerraReady.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ElementType, useEffect } from "react";
```

Comment:
In the future, we could probably re-use this component (see other [export ready component](https://github.com/clevercanary/data-explorer/blob/f4c8d8e5e24e8cc895d5928c884fbd759fbbac70/packages/data-explorer-ui/src/components/Export/components/ExportToTerraReady).